internal/component/loki/source/file/file.go

```diff
@@ -370,36 +323,33 @@ func (c *Component) createReader(opts readerOptions) (reader, error) {
 			c.IsStopping,
 		)
 		if err != nil {
-			level.Error(c.opts.Logger).Log("msg", "failed to create decompressor", "error", err, "filename", opts.path)
```

Comment:
Are we sure we want to delete this log line? Will we get the filename printed somewhere else?